seatunnel-dist/release-docs/LICENSE

```diff
@@ -836,28 +841,27 @@ The text of each license is the standard Apache 2.0 license.
      (The Apache Software License, Version 2.0) neo4j-connector-apache-spark-common (org.neo4j:neo4j-connector-apache-spark_2.12_common:4.1.0 - https://github.com/neo4j-contrib/neo4j-spark-connector/neo4j-connector-apache-spark_2.12_common)
      (The Apache Software License, Version 2.0) okhttp (com.squareup.okhttp3:okhttp:4.9.1 - https://square.github.io/okhttp/)
      (The Apache Software License, Version 2.0) okhttp-logging-interceptor (com.squareup.okhttp3:logging-interceptor:4.9.1 - https://square.github.io/okhttp/)
      (The Apache Software License, Version 2.0) org.gridkit.lab::jvm-attach-api (org.gridkit.lab:jvm-attach-api:1.5 - http://code.google.com/p/gridkit/jvm-attach-api)
-     (The Apache Software License, Version 2.0) parent-join (org.elasticsearch.plugin:parent-join-client:6.3.1 - https://github.com/elastic/elasticsearch)
-     (The Apache Software License, Version 2.0) percolator (org.elasticsearch.plugin:percolator-client:6.3.1 - https://github.com/elastic/elasticsearch)
+     (The Apache Software License, Version 2.0) parent-join (org.elasticsearch.plugin:parent-join-client:7.5.1 - https://github.com/elastic/elasticsearch)
+     (The Apache Software License, Version 2.0) percolator (org.elasticsearch.plugin:percolator-client:7.5.1 - https://github.com/elastic/elasticsearch)
      (The Apache Software License, Version 2.0) project ':json-path' (com.jayway.jsonpath:json-path:2.3.0 - https://github.com/jayway/JsonPath)
-     (The Apache Software License, Version 2.0) rank-eval (org.elasticsearch.plugin:rank-eval-client:6.3.1 - https://github.com/elastic/elasticsearch)
-     (The Apache Software License, Version 2.0) reindex (org.elasticsearch.plugin:reindex-client:6.3.1 - https://github.com/elastic/elasticsearch)
-     (The Apache Software License, Version 2.0) rest (org.elasticsearch.client:elasticsearch-rest-client:6.3.1 - https://github.com/elastic/elasticsearch)
-     (The Apache Software License, Version 2.0) rest-high-level (org.elasticsearch.client:elasticsearch-rest-high-level-client:6.3.1 - https://github.com/elastic/elasticsearch)
-     (The Apache Software License, Version 2.0) secure-sm (org.elasticsearch:elasticsearch-secure-sm:6.3.1 - https://github.com/elastic/elasticsearch)
-     (The Apache Software License, Version 2.0) server (org.elasticsearch:elasticsearch:6.3.1 - https://github.com/elastic/elasticsearch)
+     (The Apache Software License, Version 2.0) rank-eval (org.elasticsearch.plugin:rank-eval-client:7.5.1 - https://github.com/elastic/elasticsearch)
+     (The Apache Software License, Version 2.0) reindex (org.elasticsearch.plugin:reindex-client:7.5.1 - https://github.com/elastic/elasticsearch)
+     (The Apache Software License, Version 2.0) rest (org.elasticsearch.client:elasticsearch-rest-client:7.5.1 - https://github.com/elastic/elasticsearch)
```

Comment:
I find there are some CVE in `elasticsearch-rest-client:7.5.1`, maybe it's better to set a higher version.
https://mvnrepository.com/artifact/org.elasticsearch.client/elasticsearch-rest-high-level-client/7.5.1

Reply:
The reason is `flink-connector-elasticsearch7` use 7.5.1. Can't replace it.

Reply:
OK, maybe we can use a high version to override this, but this may cause incompatibility problems, it's better to directly use `flink-connector-elasticsearch7`.

Reply:
I looked at the corresponding CVE, which is due to log4j-core, but we didn't import it, so we can merge it.